docs/components/Blockquote.md

```diff
@@ -4,24 +4,4 @@
 <Blockquote>
   “The simplest scale is a single note, and sticking with a single note draws more attention to other parameters, such as rhythm and inflection.”
 </Blockquote>
-```
-
-Extends: [Base](/components/Base)
```

Comment:
Will need an update to system-docs for @rebass/components' `extend` prop. The fontSize and space props still work, this is just a regression with the docs build step